graphql/util.go

```diff
@@ -1290,3 +1290,63 @@ func getAPISubscriptionsForProject(ctx context.Context, projectId string) ([]*re
 	}
 	return res, nil
 }
+
+func GroupProjects(projects []model.ProjectRef, onlyDefaultedToRepo bool) ([]*GroupedProjects, error) {
+	groupsMap := make(map[string][]*restModel.APIProjectRef)
+
+	for _, p := range projects {
+		var groupName string
+		groupName = fmt.Sprintf("%s/%s", p.Owner, p.Repo)
+
+		if onlyDefaultedToRepo && p.RepoRefId == "" {
```

Comment:
can we check UseRepoSettings instead of RepoRefId here?

Reply:
I'm remembering now why I used RepoRefId instead of UseRepoSettings. The tests only work with RepoRefId. I can explain more. Can we maybe use RepoRefId here since it has the same affect?